api/v1alpha1/stresschaos_types.go

```diff
@@ -233,19 +232,7 @@ type Stressors struct {
 func (in *Stressors) Normalize() (string, error) {
 	stressors := ""
 	if in.MemoryStressor != nil {
-		stressors += fmt.Sprintf(" --vm %d --vm-keep", in.MemoryStressor.Workers)
-		if len(in.MemoryStressor.Size) != 0 {
-			if in.MemoryStressor.Size[len(in.MemoryStressor.Size)-1] != '%' {
-				size, err := units.FromHumanSize(in.MemoryStressor.Size)
-				if err != nil {
-					return "", err
-				}
-				stressors += fmt.Sprintf(" --vm-bytes %d", size)
-			} else {
-				stressors += fmt.Sprintf("--vm-bytes %s",
-					in.MemoryStressor.Size)
-			}
-		}
+		stressors += fmt.Sprintf(" --bigheap %d", in.MemoryStressor.Workers)
```

Comment:
Can we control how much memory it takes up?

Reply:
No. The stress-ng doesn't provide such arguments. We can only control how fast it eats up ram (but these arguments are also hard to understand and calculate).